### PR TITLE
Data tidying, bug fixes, and plot exports

### LINKED ISSUE #14: Data Last Updated
Data last updated dates on app are inconsistent

### PR DESCRIPTION
**Pull request by eroten**
- ⚠️ I organized the output data and images into appropriate folders. The paths are updated as well
- Data wrangling markdown creates small and large versions of the cases vs. viral load and variant plots. Large is 8.5x11 inches, small is 1200x675 pixels. 
- Added column names and formatting to both DTs
- Added link to this repo in footer
- Fetch and display data updated date _and_ app updated date automatically
- Add Google Analytics
- Various code styling and documentation
- Fix bug with data that only showed Omicron and Delta, not Alpha, etc. (this might have only been a local thing?)
    - Use `dplyr::full_join()` instead of `merge()`
    - No longer re-assigns object to same object name

Closes #14 , #2 , #11, #12, #7

### CODE REVIEW

File: metc-wastewater-covid-monitor/server.R [hunk 1]
```diff
@@ -287,16 +288,27 @@ server <- function(input, output) {
           buttons = c("copy", "excel", "csv"),
           searching = FALSE,
           lengthMenu = FALSE
-        )
+        ),
+        colnames = c("Date",
+                     "7-day average viral load in wastewater, M copies/person/day",
```
Comment (ashleyasmus) [inline metc-wastewater-covid-monitor/server.R]: is this correct that the load is a 7-day average, not a one-day measure? Cases are a 7-day average.

Comment (ashleyasmus) [reply metc-wastewater-covid-monitor/server.R]: Pretty sure it's a one day measure - if not, let's provide that over the 7 day aveage

File: metc-wastewater-covid-monitor/server.R [hunk 2]
```diff
@@ -294,26 +294,30 @@ server <- function(input, output) {
...
-        colnames = c("Date",
-                     "7-day average viral load in wastewater, M copies/person/day",
-                     "Standard error",
-                     "Total COVID cases",
-                     "New COVID cases",
-                     "COVID cases per capita",
-                     "7-day rolling average COVID cases per capita")
+        colnames = c(
+          "Date",
+          "7-day average viral load in wastewater, M copies/person/day",
```
Comment (ashleyasmus) [inline metc-wastewater-covid-monitor/server.R:299]: same check as before - is this really a 7 day average?

File: metc-wastewater-covid-monitor/ui.R [hunk 3]
```diff
@@ -80,12 +80,16 @@ fluidPage(
...
+  tags$div(
+    "This project is open-source. See our GitHub repository here",
     tags$a(
       href = "https://github.com/Metropolitan-Council/covid-poops",
       shiny::icon("external-link-alt", lib = "font-awesome"),
       target = "_blank"
     ),
+    tags$br(),
+    "App last updated ",
+    gh::gh("GET /repos/Metropolitan-Council/covid-poops")[49][[1]] %>% as.Date(),
```
Comment (ashleyasmus) [inline metc-wastewater-covid-monitor/ui.R:92]: oh neat

File: metc-wastewater-covid-monitor/server.R (hunk 1 above)
Comment (eroten) [reply metc-wastewater-covid-monitor/server.R]: Yes! See data-wrangle.Rmd [here](https://github.com/Metropolitan-Council/covid-poops/blob/462e33902a3661ce419361caa417f8a93443566e/data-wrangle.Rmd#L340)

Comment (ashleyasmus) [reply metc-wastewater-covid-monitor/server.R]: this is an average of multiple samples, each day - not an average across a week

**Review by ashleyasmus — APPROVED** [review]
all is well :)

**Pull request merged**